### DISCUSSION TITLE
PrimeNG version 14.2.2 not resetting slider while clicking on clear button

### DISCUSSION THREAD

**Discussion by nimishprime** [primefaces/community#1212] [PrimeNG]
We are having an issue with the p-slider inside p-columnFilter. With the filter it shows 2 buttons Clear and Apply. Clear button click is not resetting the slider to the default values. This issue is occuring with version 14, however works fine with version 11.  Either make it working like version 11 or we need a callback function to be called on click of Clear button. 

![Screenshot 2024-02-16 at 1 53 24 PM](https://github.com/primefaces/community/assets/160273611/a888eec0-ed87-48bb-a590-c1cc2f14e02a)

**Comment by cetincakiroglu** [comment]
Hi,

v14 will not receive an update in the future. It became legacy after the arrival of v17. Currently, there is only an LTS option available for v15-v16 or you can use v17. We cannot fix v14 as it is outdated.

**Reply by nimishprime** [reply]
I see the same behavior with version 17 as well where it is not resetting the slider scale on click of the Clear button. It clears the filter but scale remains as is.  


![version17_clear_button_issue](https://github.com/primefaces/community/assets/160273611/9f69096f-d696-496b-a411-83c617881c0a)
